recipe/meta.yaml

```diff
@@ -37,11 +37,13 @@ test:
     - tableschema.constraints
     - tableschema.plugins
     - tableschema.types
-  commands:
-    - tableschema --help
   requires:
     - pylama
     - tox
+    - pip
+  commands:
+    - pip check
```

Comment:
@roll, with this change we can activate automerge and let the bot merge things if all the dependencies are correct.

Reply:
@ocefap
Great! But i think i missed the message and merged before

Reply:
Im wrong it's merged also

Reply:
No problem. I checked and this PR is passing with this change, so we can activate automerge with #37